scripts/release.sh

```diff
@@ -1,9 +1,7 @@
 #!/bin/bash
 set -euo pipefail
 
-python manage.py migrate --noinput
```

Comment:
don't we want these things to happen if there *is* a database?

Reply:
maybe i could come up with a better variable, but i want to add a flag just to the review apps and have it skip these steps if present, otherwise do the normal release procedure.

i removed these for now to make sure i don't mess up the live database before i'm confident in the logic

Reply:
we're now using the presence of a `PRODUCTION` env variable to determine if we run the management commands or not.

```bash
if [ -n "${PRODUCTION}" ]; then
```